python/global_scheduler/test/test.py

```diff
@@ -119,9 +115,12 @@ def tearDown(self):
     else:
       self.p1.kill()
     # Kill local schedulers, plasma managers, and plasma stores.
-    map(subprocess.Popen.kill, self.local_scheduler_pids)
-    map(subprocess.Popen.kill, self.plasma_manager_pids)
-    map(subprocess.Popen.kill, self.plasma_store_pids)
```

Comment:
I'll have to look into this. They should be the same, but on my laptop, ` map(subprocess.Popen.kill` does not seem to kill the processes..